app/controllers/hosts_controller.rb

```diff
@@ -44,7 +44,7 @@ def index(title = nil)
     end
     respond_to do |format|
       format.html do
-        @hosts = search.includes(included_associations).paginate(:page => params[:page])
+        @hosts = search.includes(included_associations([:token])).paginate(:page => params[:page])
```

Comment:
This should be added to included_associations itself, since it'll be displayed also in the taxonomies controller's host list.